packages/changeset-bot/README.md

```diff
@@ -0,0 +1,23 @@
+# changeset-bot
```

Comment:
The big question I have is how does somebody who is not-us install and set up this bot in their own repository. I can see it works awesomely here, but I wouldn't know where to start for someone else.

Is it we run the bot, and other repos can add the bot, or is it something other than that?